packages/expo-module-scripts/templates/.npmignore

```diff
@@ -1,2 +1,4 @@
 # @generated by expo-module-scripts
 babel.config.js
+**/__tests__
+**/__mocks__
```

Comment:
Don't need `**` (the absence of `/` means any match, not only at the package root), and we want to match only directories:

```suggestion
__mocks__/
__tests__/
```

Reply:
Also, I wouldn't want to nitpick, but I think it's more common for files in `expo/expo` to have a newline at the end than not. 🙂